geo/src/lib.rs

```diff
@@ -1,4 +1,121 @@
+//! - **[`ChamberlainDuquetteArea`](algorithm::chamberlain_duquette_area::ChamberlainDuquetteArea)**: Calculate the geodesic area of a geometry
+//!
+//! ## Distance
+//!
+//! - **[`EuclideanDistance`](algorithm::euclidean_distance::EuclideanDistance)**: Calculate the euclidean distance between geometries
+//! - **[`GeodesicDistance`](algorithm::geodesic_distance::GeodesicDistance)**: Calculate the geodesic distance between geometries using the algorithm presented in _Algorithms for geodesics_ by Charles Karney (2013)
+//! - **[`HaversineDistance`](algorithm::haversine_distance::HaversineDistance)**: Calculate the geodesic distance between geometries using the haversine formula
+//! - **[`VincentyDistance`](algorithm::vincenty_distance::VincentyDistance)**: Calculate the geodesic distance between geometries using Vincenty’s formula
+//!
+//! ## Length
+//!
+//! - **[`EuclideanLength`](algorithm::euclidean_length::EuclideanLength)**: Calculate the euclidean length of a geometry
+//! - **[`GeodesicLength`](algorithm::geodesic_length::GeodesicLength)**: Calculate the geodesic length of a geometry using the algorithm presented in _Algorithms for geodesics_ by Charles Karney (2013)
+//! - **[`HaversineLength`](algorithm::haversine_length::HaversineLength)**: Calculate the geodesic length of a geometry using the haversine formula
+//! - **[`VincentyLength`](algorithm::vincenty_length::VincentyLength)**: Calculate the geodesic length of a geometry using Vincenty’s formula
+//!
+//! ## Simplify
+//!
+//! - **[`Simplify`](algorithm::simplify::Simplify)**: Simplify a geometry using the Ramer–Douglas–Peucker algorithm
+//! - **[`SimplifyIdx`](algorithm::simplify::SimplifyIdx)**:
+//! - **[`SimplifyVW`](algorithm::simplifyvw::SimplifyVW)**: Simplify a geometry using the Visvalingam-Whyatt algorithm
+//! - **[`SimplifyVWPreserve`](algorithm::simplifyvw::SimplifyVWPreserve)**: Simplify a geometry with the using a topology-preserving variant of the Visvalingam-Whyatt algorithm
+//! - **[`SimplifyVwIdx`](algorithm::simplifyvw::SimplifyVwIdx)**:
+//!
+//! ## Query
+//!
+//! - **[`Bearing`](algorithm::bearing::Bearing)**:
+//! - **[`ClosestPoint`](algorithm::closest_point::ClosestPoint)**:
+//! - **[`Contains`](algorithm::contains::Contains)**:
+//! - **[`CoordinatePosition`](algorithm::coordinate_position::CoordinatePosition)**:
+//! - **[`Intersects`](algorithm::intersects::Intersects)**:
+//! - **[`IsConvex`](algorithm::is_convex::IsConvex)**:
+//! - **[`LineInterpolatePoint`](algorithm::line_interpolate_point::LineInterpolatePoint)**:
+//! - **[`LineLocatePoint`](algorithm::line_locate_point::LineLocatePoint)**:
+//!
+//! ## Similarity
+//!
+//! - **[`FrechetDistance`](algorithm::frechet_distance::FrechetDistance)**: Calculate the similarity between [`LineString`]s using the Fréchet distance
+//!
+//! ## Winding
+//!
+//! - **[`Orient`](algorithm::orient::Orient)**: Orients the exterior and interior rings of a [`Polygon`] according to convention
```

Comment:
We should probably clarify what convention means here for non-GIS users

Reply:
Should we clarify that here or in the `Orient` trait docs?

Reply:
How about something like:

> Apply a specified [`Winding`] to a [`Polygon`]'s interior and exterior rings.

I realize that you just pulled this doc from the existing `Orient` docs, and so it's kind of unfair to put you on the hook for also improving all the docs just because you decided to helpfully consolidate them... so feel free to take or leave this suggestion.

Reply:
Addressed in https://github.com/georust/geo/pull/619/commits/54160329232ae16c0b9fa1aac1f40d96ef27d1f4